<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>1024</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="widget1" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>1024</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(:/ui/QR.png)</string>
   </property>
   <widget class="QLabel" name="label1">
    <property name="geometry">
     <rect>
      <x>152</x>
      <y>331</y>
      <width>296</width>
      <height>296</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget4" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>1024</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(:/ui/main.png)</string>
   </property>
   <widget class="QLabel" name="label4_1">
    <property name="geometry">
     <rect>
      <x>208</x>
      <y>69</y>
      <width>188</width>
      <height>140</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label4_2">
    <property name="geometry">
     <rect>
      <x>208</x>
      <y>221</y>
      <width>188</width>
      <height>140</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label4_4">
    <property name="geometry">
     <rect>
      <x>403</x>
      <y>69</y>
      <width>188</width>
      <height>140</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label4_3">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>69</y>
      <width>188</width>
      <height>140</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label4_5_1">
    <property name="geometry">
     <rect>
      <x>55</x>
      <y>667</y>
      <width>90</width>
      <height>89</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(:/image/ui/r1.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label4_5_2">
    <property name="geometry">
     <rect>
      <x>191</x>
      <y>667</y>
      <width>90</width>
      <height>89</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(:/image/ui/r2.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label4_5_3">
    <property name="geometry">
     <rect>
      <x>336</x>
      <y>667</y>
      <width>72</width>
      <height>89</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(:/image/ui/r3.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label4_5_4">
    <property name="geometry">
     <rect>
      <x>472</x>
      <y>667</y>
      <width>72</width>
      <height>89</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(:/image/ui/r4.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label4_5_5">
    <property name="geometry">
     <rect>
      <x>55</x>
      <y>786</y>
      <width>90</width>
      <height>89</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(:/image/ui/r5.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label4_5_6">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>786</y>
      <width>72</width>
      <height>89</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(:/image/ui/r6.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label4_5_7">
    <property name="geometry">
     <rect>
      <x>336</x>
      <y>786</y>
      <width>72</width>
      <height>89</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(:/image/ui/r7.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label4_5_8">
    <property name="geometry">
     <rect>
      <x>472</x>
      <y>786</y>
      <width>72</width>
      <height>89</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(:/image/ui/r8.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label4_5_1_1">
    <property name="geometry">
     <rect>
      <x>55</x>
      <y>667</y>
      <width>90</width>
      <height>89</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(:/image/ui/w1.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label4_5_2_1">
    <property name="geometry">
     <rect>
      <x>191</x>
      <y>667</y>
      <width>90</width>
      <height>89</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(:/image/ui/w2.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label4_5_3_1">
    <property name="geometry">
     <rect>
      <x>336</x>
      <y>667</y>
      <width>72</width>
      <height>89</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(:/image/ui/w3.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label4_5_4_1">
    <property name="geometry">
     <rect>
      <x>472</x>
      <y>667</y>
      <width>72</width>
      <height>89</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(:/image/ui/w4.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label4_5_5_1">
    <property name="geometry">
     <rect>
      <x>55</x>
      <y>786</y>
      <width>90</width>
      <height>89</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(:/image/ui/w5.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label4_5_6_1">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>786</y>
      <width>72</width>
      <height>89</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(:/image/ui/w6.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label4_5_7_1">
    <property name="geometry">
     <rect>
      <x>336</x>
      <y>786</y>
      <width>72</width>
      <height>89</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(:/image/ui/w7.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label4_5_8_1">
    <property name="geometry">
     <rect>
      <x>472</x>
      <y>786</y>
      <width>72</width>
      <height>89</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(:/image/ui/w8.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label4_6_1">
    <property name="geometry">
     <rect>
      <x>118</x>
      <y>482</y>
      <width>108</width>
      <height>108</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label4_6_2">
    <property name="geometry">
     <rect>
      <x>271</x>
      <y>482</y>
      <width>120</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label4_6_3">
    <property name="geometry">
     <rect>
      <x>271</x>
      <y>522</y>
      <width>120</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label4_6_4">
    <property name="geometry">
     <rect>
      <x>271</x>
      <y>562</y>
      <width>120</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label4_6_5">
    <property name="geometry">
     <rect>
      <x>392</x>
      <y>482</y>
      <width>120</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label4_6_6">
    <property name="geometry">
     <rect>
      <x>392</x>
      <y>522</y>
      <width>120</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label4_6_7">
    <property name="geometry">
     <rect>
      <x>392</x>
      <y>562</y>
      <width>120</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget3" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>1024</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(:/ui/succees_face.png)</string>
   </property>
   <widget class="QLabel" name="label3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>296</width>
      <height>296</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget2" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>1024</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(:/ui/start_face.png)</string>
   </property>
   <widget class="QLabel" name="label2">
    <property name="geometry">
     <rect>
      <x>84</x>
      <y>242</y>
      <width>432</width>
      <height>432</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget0" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>1024</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(:/ui/bind.png)</string>
   </property>
   <widget class="QLabel" name="label0">
    <property name="geometry">
     <rect>
      <x>152</x>
      <y>331</y>
      <width>296</width>
      <height>296</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 40pt &quot;Ubuntu&quot;;
background-image: url(:/ui/b_i.png);
background-color: qlineargradient(spread:pad, x1:0.0457746, y1:0.062, x2:1, y2:1, stop:0 rgba(6, 87, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
